content/manuals/build/concepts/dockerfile.md

```diff
@@ -164,23 +149,17 @@ the container.
 
 ### Comments
 
-Note the `# install app dependencies` line. This is a comment. Comments in
-Dockerfiles begin with the `#` symbol. As your Dockerfile evolves, comments can
-be instrumental to document how your Dockerfile works for any future readers
-and editors of the file, including your future self!
+The `# install app dependencies` line is a comment. Comments in Dockerfiles start
+with `#`. Comments help document your Dockerfile for future readers and editors.
 
 > [!NOTE]
->
-> You might've noticed that comments are denoted using the same symbol as the
-> [syntax directive](#dockerfile-syntax) on the first line of the file.
-> The symbol is only interpreted as a directive if the pattern matches a
-> directive and appears at the beginning of the Dockerfile. Otherwise, it's
-> treated as a comment.
+> Comments use the same symbol as the [syntax directive](#dockerfile-syntax) on the
+> first line. The symbol is only a directive if it matches a directive pattern and is
+> at the start of the Dockerfile. Otherwise, it is a comment.
 
 ### Installing dependencies
 
-The second `RUN` instruction installs the `flask` dependency required by the
-Python application.
+The second `RUN` instruction installs the `flask` dependency for the Python app.
```

Comment:
Why is "app" better than "application"?

Reply:
Most of our guides refer to applications as app. We can harmonize either way.